examples/deployment/scripts/transfer/multisig_sc.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * Description:
+ * This file demonstrates the example to
+   - load a signed lsig from assets and use that lsig to validate transactions
+   - create a signed lsig using sdk and use that lsig to validate transactions
+*/
+const { transferAlgo } = require("./common");
+const { transferMicroAlgos } = require("algob");
+const algosdk = require('algosdk');
+
+async function run(runtimeEnv, deployer) {
+  const goldOwnerAccount = deployer.accountsByName.get("gold-owner-account");
+  const johnAccount = deployer.accountsByName.get("john-account");
+  const bobAccount = deployer.accountsByName.get("bob-account");
+
+  //Get Logic Signature
+  const logicSignature = await deployer.getLogicSignature("3-gold-delegated-asc.teal", []);
+
+  /*
+    These accounts are generated by using goal (goal account new)
+    The above example demonstrates a transaction validated by a smart contract signed by a multisig account with threshold of 2
+    User saves the signed logic signature file in assets/ which is extracted and used here.
+  */
```

Comment:
we should remove this example, as it uses `goal` , we can support multisignature using SDK too,

and using `goal` and `SDK` is confusing for user,